api/proto/info/info.proto

```diff
@@ -66,6 +66,7 @@ enum ZInfoTypes {
   ZiContentTree = 8;
   ZiBlobList = 9;
   ZiAppInstMetaData = 10;
+  Z_INFO_TYPES_HARDWARE = 11;
```

Comment:
I know this is what yetus says, but it is more important to me that the enum values in a particular enum follow the same style. Thus ZiHardware makes more sense.

Reply:
corrected